sdk/highlight-ruby/highlight/Gemfile.lock

```diff
@@ -230,10 +231,11 @@ GEM
       parser (>= 3.2.1.0)
     ruby-progressbar (1.13.0)
     ruby2_keywords (0.0.5)
+    strscan (3.1.0)
     unicode-display_width (2.4.2)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
is this change ok? not sure if this affects cross-platform compatibilty

Reply:
weird that dependabot did that... let me revert